config.go

```diff
@@ -20,7 +23,11 @@ type HystrixCommandConfig struct {
 }
 
 // NewHystrixConfig should be used to give hystrix commandName and config
-func NewHystrixConfig(commandName string, commandConfig HystrixCommandConfig) HystrixConfig {
+func NewHystrixConfig(commandName string, commandConfig HystrixCommandConfig, f ...fallbackFunction) HystrixConfig {
```

Comment:
Wouldn't it be better to add the fallback function as an attribute of `HystrixCommandConfig`?

Reply:
it would be but `HystrixCommandConfig` is of type `hystrix.CommandConfig` 
```
type CommandConfig struct {
	Timeout                int `json:"timeout"`
	MaxConcurrentRequests  int `json:"max_concurrent_requests"`
	RequestVolumeThreshold int `json:"request_volume_threshold"`
	SleepWindow            int `json:"sleep_window"`
	ErrorPercentThreshold  int `json:"error_percent_threshold"`
}
```

Reply:
Isn't `hystrix.CommandConfig` constructed manually from the attributes of `HystrixCommandConfig`? In that case adding another attribute would not matter. The constructor would then return something like : 
```go
return HystrixConfig{
		commandName: commandName,
		commandConfig: hystrix.CommandConfig{
			Timeout:                commandConfig.Timeout,
			MaxConcurrentRequests:  commandConfig.MaxConcurrentRequests,
			RequestVolumeThreshold: commandConfig.RequestVolumeThreshold,
			SleepWindow:            commandConfig.SleepWindow,
			ErrorPercentThreshold:  commandConfig.ErrorPercentThreshold,
		},
                fallbackFn : commandConfig.fallback
	}
```

Another alternative would be just to make it a single argument without the spread operator. Using `...` without ever expecting more than one argument of that type isn't idiomatic

Reply:
👍 to making it part of `HystrixConfig`